homeassistant/components/nintendo_parental/strings.json

```diff
@@ -28,11 +28,19 @@
       "time_remaining": {
         "name": "Screen time remaining"
       }
+    },
+    "time": {
+      "bedtime_alarm": {
+        "name": "Bedtime alarm"
+      }
     }
   },
   "exceptions": {
     "auth_expired": {
       "message": "Authentication expired. Please remove and re-add the integration to reconnect."
+    },
```

Comment:
Not directly related to this PR, but I think a reauth flow should be prioritized over adding new functionality.